x-pack/plugins/synthetics/server/synthetics_service/formatters/private_formatters/formatters.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { HTTPFormatMap, httpFormatters } from './http_formatters';
+import { TCPFormatMap, tcpFormatters } from './tcp_formatters';
+import { ICMPFormatMap, icmpFormatters } from './icmp_formatters';
+import { BrowserFormatMap, browserFormatters } from './browser_formatters';
+import { CommonFormatMap, commonFormatters } from './common_formatters';
+
+type Formatters = HTTPFormatMap & TCPFormatMap & ICMPFormatMap & BrowserFormatMap & CommonFormatMap;
```

Comment:
I'm not sure if this is how we had things previously, but it seems really brittle. (I'm sure I probably wrote this).

I don't like how all the formatters are spread together instead of being separated out by monitor type

It seems like at least, common should come first, and then the specific monitor type formatters should overwrite things.